ultimate_cron.module

```diff
@@ -608,14 +237,14 @@ function ultimate_cron_get_hook($name, $reset = FALSE) {
  */
 function ultimate_cron_get_module_hooks($module) {
   $items = array();
-  if (module_hook($module, 'cronapi')) {
-    $items = module_invoke($module, 'cronapi', NULL);
+  if (Drupal::moduleHandler()->implementsHook($module, 'cronapi')) {
```

Comment:
As discussed, open a task "Replace hook_cronapi() with default configuration, test it" if not already done.